celery_serverless/watchdog.py

```diff
@@ -55,6 +60,15 @@ def get_queue_length(self):
         logger.debug('_watched reported %s jobs awaiting', len_)
         return len_
 
+    def is_shutdown_requested(self):
+        if hasattr(self._intercom, 'get_shutdown_requested'):
+            return self._intercom.get_shutdown_requested(self._shutdown_key)
+        return self._get_shutdown_requested(self._shutdown_key)
```

Comment:
Is there a reason why we prefer `self._intercom.get_shutdown_requested(self._shutdown_key)` over `self._get_shutdown_requested(self._shutdown_key)`? Both perform the lookup on the intercom, don't they?

Reply:
I am getting to a point that the Intercom should be an interface, not a capsule to StrictRedis.

I mean, a refector to allow other Intercoms should occur soon